app/models/middleware_server_performance.rb

```diff
@@ -16,7 +16,11 @@ def self.build_results_for_report_middleware(options)
     start_time, end_time, interval = parse_time_interval(options)
     results = []
     MiddlewareServer.find_each do |ms|
-      raw_stats = fetch_raw_stats(ms, start_time, end_time, interval)
+      begin
+        raw_stats = fetch_raw_stats(ms, start_time, end_time, interval)
```

Comment:
thanks @jrafanie. @josejulio proposed something similar to your suggestion [in the gem ](https://github.com/hawkular/hawkular-client-ruby/pull/115) by introducing a specific exception. cc @pilhuhn 
we can proceed in that direction and handle here only one type of errors if there's a consensus on this approach.
@pilhuhn @josejulio wdyt?